docs/data/sql_functions.yml

```diff
@@ -516,7 +516,14 @@ conditional:
   - sql: NULLIF(value1, value2)
     description: Returns NULL if value1 is equal to value2; returns value1 otherwise. E.g., NULLIF(5, 5) returns NULL; NULLIF(5, 0) returns 5.
   - sql: COALESCE(value1, value2 [, value3]*)
```

Comment:
The syntax here seems to be incorrect as it suggests a required minimum of two arguments, but it's only one argument.